pkg/agent/controller/networkpolicy/status_controller.go

```diff
@@ -82,6 +88,22 @@ func realizedRulePolicyIndexFunc(obj interface{}) ([]string, error) {
 	return []string{string(rule.policyID)}, nil
 }
 
+// invalidRule is the struct kept by StatusController for storing an invalid rule reported by antrea-agent.
+type invalidRule struct {
+	RuleID   string `json:"rule_id"`
```

Comment:
I may be wrong, but I think we typically use camelCase for json field names in our code base / APIs?

Reply:
It should be camelCase.